# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/performance-timeline/idlharness.any.js.ini
# Commit: e04ef5171459
# Full Hash: e04ef517145991e2aee797e935f43b7be35ff7aa
# Author: Will Hawkins <whawkins@mozilla.com>
# Date: 2019-04-09 09:54:40
# Regressor Bug: 1539006
# File Overlap Count: 1
# Description:
#   Bug 1539006: Properly implement Performance Timeline Level 2 w3c spec. r=baku
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D25426
# ==============================================================================

diff -r ffaff6fb1fe3 -r e04ef5171459 testing/web-platform/meta/performance-timeline/idlharness.any.js.ini
--- a/testing/web-platform/meta/performance-timeline/idlharness.any.js.ini	Mon Apr 08 19:00:31 2019 +0000
+++ b/testing/web-platform/meta/performance-timeline/idlharness.any.js.ini	Tue Apr 09 00:43:10 2019 +0000
@@ -2,7 +2,10 @@
   expected: TIMEOUT
 
 [idlharness.any.serviceworker.html]
-  [PerformanceObserver interface: attribute supportedEntryTypes]
+  [PerformanceObserver interface: calling observe(PerformanceObserverInit) on observer with too few arguments must throw TypeError]
+    expected: FAIL
+  
+  [PerformanceObserver interface: operation observe(PerformanceObserverInit)]
     expected: FAIL
 
   [PerformanceMark interface: attribute detail]
@@ -16,10 +19,13 @@
 
 
 [idlharness.any.sharedworker.html]
-  [PerformanceObserver interface: attribute supportedEntryTypes]
+  [PerformanceMark interface: attribute detail]
     expected: FAIL
 
-  [PerformanceMark interface: attribute detail]
+  [PerformanceObserver interface: calling observe(PerformanceObserverInit) on observer with too few arguments must throw TypeError]
+    expected: FAIL
+  
+  [PerformanceObserver interface: operation observe(PerformanceObserverInit)]
     expected: FAIL
 
   [PerformanceMark interface: mark must inherit property "detail" with the proper type]
@@ -30,10 +36,13 @@
 
 
 [idlharness.any.html]
-  [PerformanceObserver interface: attribute supportedEntryTypes]
+  [PerformanceMark interface: attribute detail]
     expected: FAIL
 
-  [PerformanceMark interface: attribute detail]
+  [PerformanceObserver interface: calling observe(PerformanceObserverInit) on observer with too few arguments must throw TypeError]
+    expected: FAIL
+  
+  [PerformanceObserver interface: operation observe(PerformanceObserverInit)]
     expected: FAIL
 
   [PerformanceMark interface: mark must inherit property "detail" with the proper type]
@@ -44,10 +53,13 @@
 
 
 [idlharness.any.worker.html]
-  [PerformanceObserver interface: attribute supportedEntryTypes]
+  [PerformanceMark interface: attribute detail]
     expected: FAIL
 
-  [PerformanceMark interface: attribute detail]
+  [PerformanceObserver interface: calling observe(PerformanceObserverInit) on observer with too few arguments must throw TypeError]
+    expected: FAIL
+  
+  [PerformanceObserver interface: operation observe(PerformanceObserverInit)]
     expected: FAIL
 
   [PerformanceMark interface: mark must inherit property "detail" with the proper type]